cpp/ql/test/library-tests/ir/ir/PrintAST.expected

```diff
@@ -6627,3 +6627,23 @@ ir.cpp:
 #  988|                 Type = double
 #  988|                 Value = 1.0
 #  988|                 ValueCategory = prvalue
+#  991| int ExprStmt(int, int, int)
+#  991|   params: 
+#  991|     0: b
+#  991|         Type = int
+#  991|     1: y
+#  991|         Type = int
+#  991|     2: z
+#  991|         Type = int
+#  991|   body: { ... }
+#  992|     0: declaration
+#  992|       0: definition of x
+#  992|           Type = int
+#  992|         init: initializer for x
+#  992|           expr: (statement expression)
+#  992|               Type = int
+#  992|               ValueCategory = prvalue
+# 1002|     1: return ...
+# 1002|       0: (statement expression)
+# 1002|           Type = int
+# 1002|           ValueCategory = prvalue
```

Comment:
It looks like `PrintAST` doesn't support `StmtExpr`. That's not a blocker for this PR.